packages/netlify-cms-widget-relation/src/RelationControl.js

```diff
@@ -11,17 +11,18 @@ import { FixedSizeList } from 'react-window';
 const Option = ({ index, style, data }) => <div style={style}>{data.options[index]}</div>;
 
 const MenuList = props => {
-  if (props.isLoading || props.options.length <= 0) {
+  if (props.isLoading || props.options.length <= 0 || !Array.isArray(props.children)) {
```

Comment:
In case there are options and they are all selected, `props.children` is not an array, but a component that displays "No options"